servers/src/main/java/tachyon/worker/block/BlockDataManager.java

```diff
@@ -430,6 +430,27 @@ public void userHeartbeat(long userId, List<Long> metrics) {
   }
 
   /**
+   * Gets the metadata of a block given its blockId or throws IOException.
```

Comment:
Could you document this in more detail (ie. how its different from `getBlockMeta`)?